README.md

```diff
@@ -14,99 +12,88 @@ Check out our `examples` for a few highlights and use cases of Phoenics.
 
 | Example | Link |
 |:--------|:-----|
-| Sequential optimization           |  [examples/optimization_sequential](https://github.com/chemos-inc/phoenics/tree/master/examples/optimization_sequential)  |
-
-
+| Sequential optimization           |  [examples/optimization_sequential](https://github.com/atinary-technologies/phoenics/tree/master/examples/optimization_sequential)  |
 
-### Getting started
 
-Phoenics is developed for Linux based systems. We also support MacOS and Windows, but support for these platforms is experimental at the moment. Although we hope that you will not encounter any issues we would appreciate if you contacted us should you run into any of them.
 
-###### Option 1 (recommended)
-Phoenics can be installed for all three platforms via pip
-```
-pip install phoenics 
-```
+## Getting started
 
-###### Option 2
-For Linux systems, Phoenics is also available through anaconda
-```
-conda install -c chemos phoenics
-```
+Phoenics is developed for Linux based systems. Although we hope that you will not encounter any issues we would appreciate if you contacted us should you run into any of them.
```

Comment:
Phoenics is developed for Linux based systems. Please contact us should you run into issues.